src/components/charts/SummaryPieChart.jsx

```diff
@@ -0,0 +1,61 @@
+
+import {
+  ResponsiveContainer, PieChart, Pie, Legend, Tooltip, Cell,
+} from 'recharts';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { localTheme } from '../../localconf';
+
+const pieChartContainerStyle = {
+  height: '200px',
+  width: '200px',
+};
+
+const pieChartStyle = {
```

Comment:
This and `pieStyle` aren't being used anywhere, do we need them?

Reply:
Yes I just leave this for style improving, will change according to SPEC.